pkg/didcomm/protocol/introduce/service.go

```diff
@@ -85,19 +85,20 @@ type Service struct {
 	store       storage.Store
 	callbacks   chan *metaData
 	ctx         internalContext
+	wg          sync.WaitGroup
 	stop        chan struct{}
 	closedMutex sync.Mutex
 	closed      bool
 }
 
-// provider contains dependencies for the DID exchange protocol and is typically created by using aries.Context()
-type provider interface {
+// Provider contains dependencies for the DID exchange protocol and is typically created by using aries.Context()
+type Provider interface {
```

Comment:
Does this need to be exposed for gomocks ? just trying to understand the reason for this change.

Reply:
Yep